src/routes/activities.js

```diff
@@ -52,5 +52,16 @@ exports.plugin = {
       },
       handler: addCard,
     });
+
+    server.route({
+      method: 'DELETE',
+      path: '/card/{cardId}',
```

Comment:
même si c'est pas la route en elle même ? (la route ça reste /activities) genre courses/user on a user au singulier parce qu'on veut qu'un user, là on veut supprimer qu'une seule carte c'est pas pareil ?

Reply:
Normalement oui (tu peux regarder dans customer, il y a des cas comme ca). J'ai trouvé un article qui parle des bonnes pratiques de rest et notamment du nommage : https://stackoverflow.blog/2020/03/02/best-practices-for-rest-api-design/ "Use nouns instead of verbs in endpoint paths" et les deux paragraphes d'apres. 

La route courses/user etait un peu differente (et on peut se poser la question d'ailleurs d'ajouter un s) mais le 'user' ne faisait pas vraiment reference a un modele mais plutot au fait qu'on voulait les infos accessibles a un utilisateur en particulier par rapport a une formation.

Reply:
done